python/bayesmixpy/run.py

```diff
@@ -0,0 +1,112 @@
+import os
+import shutil
+import subprocess
+import numpy as np
+
+from tempfile import TemporaryDirectory
+from pathlib import Path
+
+
+BAYESMIX_EXE = os.environ["BAYESMIX_EXE"]
+RUN_CMD = BAYESMIX_EXE + " {0} {1} {2} {3} {4} {5} {6} {7} {8} {9} {10}"
+
+
+def _is_file(a: str):
+    p = Path(a)
+    return p.exists() and p.is_file()
+
+
+def _maybe_print_to_file(maybe_proto: str, proto_name: str, out_dir: str):
+    """If maybe_proto is a file, returns the file name.
+    If maybe_proto is a string representing a message, prints the message to
+    a file and returns the file name.
+    """
```

Comment:
Given this behavior, we should give default values (`None`?) to the last two arguments